### DISCUSSION TITLE
Hearing aids

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#975] [General]
Does someone have them?
What model and where do you order hearing supplies for them?

**Comment by MirkosMangle** [comment]
Costco hearing aids are the best options for the money

**Comment by hernessteve** [comment]
Both of my parents have hearing aids and i usually help them to order accessories for them and/or to find information about some new aids.  So far i learnt that you can get much better price if you decide to [buy hearing aids online](https://audiologyisland.com/hearing-aids/). The model and brand - that is very individual and it is better if you visit an audiologist and make a decision together with him.
